src/sdk/strategy/cbd-strategy.ts

```diff
@@ -66,18 +62,26 @@ export class CbdStrategy {
 
 export class DeployedCbdStrategy {
   private constructor(
-    public readonly decrypter: CbdTDecDecrypter,
+    public readonly decrypter: ThresholdDecrypter,
```

Comment:
Ignoring for now, but we need some kind of common naming scheme for the Strategy and its encrypter/decrypter.

`DeployedCbdStrategy` having a `ThresholdDecrypter` seems a bit disjointed/disconnected - we need to tie the strategy and encrypter/decrypter together through common naming somehow. I know this is a hot-button topic across the team, so no recommendations from my end for this PR, but something we could collectively discuss and address separately.